test/test_collection.py

```diff
@@ -2253,6 +2253,35 @@ def afind(*args, **kwargs):
         for helper, args in helpers:
             helper(*args, let={})  # type: ignore
 
+    def assertRepr(self, obj):
+        new_obj = eval(repr(obj))
+        self.assertEqual(type(new_obj), type(obj))
+        self.assertEqual(repr(new_obj), repr(obj))
+
+    def test_index_model_repr(self):
+        self.assertRepr(IndexModel("hello"))
+        self.assertRepr(IndexModel([("hello", DESCENDING), ("world", ASCENDING)]))
+        self.assertRepr(IndexModel([("hello", DESCENDING), ("world", ASCENDING)], name="hello_world"))
+
```

Comment:
If there is some grouping to the tests such that a blank line is warranted, I'd use a comment to explain rather than a blank line (or even separate tests).